### PR TITLE
Fix for bug in view_users() and edit_users()

### PR DESCRIPTION
**Pull request by pkdash**
Due to this bug, the user as a member of a deleted group was not seeing the access control panel even if the user had explicit access on that resource.

### CODE REVIEW

**Comment by pkdash** [comment]
@alvacouch can you review this small change? I am not sure if similar changes are still needed where ```.exclude()``` has been used.

File: hs_access_control/models.py [hunk 1]
```diff
@@ -2298,7 +2298,8 @@ def view_users(self):
                                       u2urp__privilege__lte=PrivilegeCodes.VIEW) |
                                     Q(u2ugp__group__g2grp__resource=self.resource,
                                       u2ugp__group__g2grp__privilege__lte=PrivilegeCodes.VIEW)))\
-                           .exclude(u2ugp__group__gaccess__active=False).distinct()
+                           .exclude(Q(u2ugp__group__gaccess__active=False,
```
Comment (alvacouch) [inline hs_access_control/models.py]: Just move u2ugp__group__gaccess__active=True in the upper phrase.
Not sure why it's like this, but it serves no purpose.  Expression will be
```
return User.objects.filter(Q(is_active=True) &
                            (Q(u2urp__resource=self.resource,
                               u2urp__privilege__lte=PrivilegeCodes.CHANGE) |
                             Q(u2ugp__group__gaccess__active=True, 
                               u2ugp__group__g2grp__resource=self.resource,
                               u2ugp__group__g2grp__privilege__lte=PrivilegeCodes.CHANGE))
                       .distinct()
```

File: hs_access_control/models.py [hunk 2]
```diff
@@ -2319,7 +2320,8 @@ def edit_users(self):
                                   u2urp__privilege__lte=PrivilegeCodes.CHANGE) |
                                 Q(u2ugp__group__g2grp__resource=self.resource,
                                   u2ugp__group__g2grp__privilege__lte=PrivilegeCodes.CHANGE)))\
-                       .exclude(u2ugp__group__gaccess__active=False).distinct()
+                       .exclude(Q(u2ugp__group__gaccess__active=False,
```
Comment (alvacouch) [inline hs_access_control/models.py]: Same comment: remove the exclude, add the qualification to the upper query.

**Comment by alvacouch** [comment]
There are identical problems in edit_resources and view_resources.

**Comment by alvacouch** [comment]
@pkdash review completed.

**Comment by pkdash** [comment]
@alvacouch A unit test was failing which I had to fix. Can you review this latest change?

**Comment by alvacouch** [comment]
@pkdash See comments above for a complete list of changes required, including view_resources and edit_resources. I have checked and all other uses of exclude are valid.  **This can wait.** Let's fix the bug.

**Comment by alvacouch** [comment]
@pkdash I approve this as a hotfix.

**Pull request merged**